packages/SwingSet/test/bootstrap-relay.js

```diff
@@ -50,12 +50,13 @@ export const buildRootObject = () => {
     },
 
     upgradeVat: async ({ name, bundleCapName, vatParameters = {} }) => {
+      /** @type {{ adminNode: import('@agoric/swingset-vat').VatAdminFacet, result: import('../src/types-external.js').VatRestartResults }} */
       const vat = vatData.get(name) || Fail`unknown vat name: ${q(name)}`;
       const bcap = await E(vatAdmin).getNamedBundleCap(bundleCapName);
       const options = { vatParameters };
-      const incarnationNumber = await E(vat.adminNode).upgrade(bcap, options);
-      vat.incarnationNumber = incarnationNumber;
-      return incarnationNumber;
+      const result = await E(vat.adminNode).restartVat(bcap, options);
+      vat.result = result;
+      return result;
```

Comment:
It probably doesn't matter too much since this is just a test, but this should be left as `incarnationNumber`.

Reply:
`restartVat` (née `upgrade) returns a result object though: `{ incarnationNumber: number }`. https://github.com/Agoric/agoric-sdk/blob/8ab381fc38b55ef77873f2a2c4a46eff88bbf9e4/packages/SwingSet/src/vats/vat-admin/vat-vat-admin.js#L537

This PR doesn't actually change the behavior, just renames (and type annotates) to avoid confusion.

Reply:
Ooh, you're right.  I missed that it's wrapped in an object.